Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/RevitReceiveBinding.cs

```diff
@@ -55,7 +59,7 @@ public async Task Receive(string modelCardId)
       // Receive host objects
       HostObjectBuilderResult conversionResults = await unitOfWork
         .Service.Execute(
-          modelCard.GetReceiveInfo(),
+          modelCard.GetReceiveInfo(_revitSettings.HostSlug.NotNull()),
```

Comment:
Is there a chance the hostSlug would not be null? 🤔

Reply:
No it should never be null, you're right.
`HostSlug` is not nullable... I just copied this block from the send bindings.

However, likley we should refactor these settings to share a common interface, since right now all connector settings look wildly different from one another despite basically just storing the host app version slugs.

Reply:
https://spockle.atlassian.net/browse/DUI3-484